ajax/libs/query-string/package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "query-string",
+  "filename": "index.min.js",
```

```suggestion
  "filename": "index.js",
```
Minified file doesn't not exist in the package or auto-update config.